seed/views/v3/columns.py

```diff
@@ -149,6 +154,68 @@ def update(self, request, pk=None):
                 .update(comstock_mapping=None)
         return super(ColumnViewSet, self).update(request, pk)
 
+    @ajax_request_class
+    @has_perm_class('can_modify_data')
+    def destroy(self, request, pk=None):
+        org_id = self.get_organization(request)
+        try:
+            column = Column.objects.get(id=pk, organization_id=org_id)
+        except Column.DoesNotExist:
+            return JsonResponse({
+                'success': False,
+                'message': 'Cannot find column in org=%s with pk=%s' % (org_id, pk)
+            }, status=status.HTTP_404_NOT_FOUND)
+
+        if not column.is_extra_data:
+            return JsonResponse({
+                'success': False,
+                'message': 'Only extra_data columns can be deleted'
+            }, status=status.HTTP_400_BAD_REQUEST)
+
+        # Delete key from jsonb data
+        if column.table_name == 'PropertyState':
+            states = PropertyState.objects.filter(organization_id=org_id, data_state=DATA_STATE_MATCHING,
+                                                  extra_data__has_key=column.column_name)
```

Comment:
In most cases, this query worked and this feature seems to work as expected. 

But in testing with an old prod database that's been upgraded to latest develop, I ran into one issue with this query:
```
django.db.utils.ProgrammingError: operator does not exist: json ? unknown
LINE 1: ..._state" = 3 AND "seed_propertystate"."extra_data" ? 'Email' ...
                                                             ^
HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
```
The only relevant difference I found between this DB compared to others is that the `extra_data` field type is json (vs jsonb). Given the error, it seems that this would be the cause.

I'm still doing research into what would've caused my current DB to type this field as json vs jsonb as well as the extra_data field type within a non-testing copy of the current prod DB.